app/templates/buyers/_base_brief_summary_page.html

```diff
@@ -57,17 +57,38 @@
                 Optional
               {% endcall %}
             {% elif not question.answer_required %}
-              {{ summary.edit_link("Edit", url_for("buyers.edit_brief_submission", framework_slug=framework.slug, lot_slug=brief_data.lot, brief_id=brief_id, section_id=question.section_id, _anchor=question.id)) }}
+              {{ summary.edit_link("Edit", url_for("buyers.edit_brief_submission", framework_slug=framework.slug, lot_slug=brief_data.lot, brief_id=brief_data.id, section_id=question.section_id, _anchor=question.id)) }}
             {% else %}
               {% call summary.field() %}{% endcall %}
             {% endif %}
           {% endif %}
-        
+
+        {% endcall %}
+      {% endcall %}
+    </div>
+
+    {% if brief_data.status == "live": %}
+    <div class="column-one-whole">
+
+      {{ summary.heading("Clarification questions", id="clarification-questions") }}
+      {% call(question) summary.list_table(
+        brief_data.clarificationQuestions,
+        field_headings=[
```

Comment:
Can these headings come from the question content at all?  Or probably not worth it?